webtransport/README.md

```diff
@@ -0,0 +1,52 @@
+# libp2p WebTransport
+
+| Lifecycle Stage | Maturity                 | Status | Latest Revision |
+|-----------------|--------------------------|--------|-----------------|
+| 1A              | DRAFT                    | Active | r0, 2022-04-03  |
```

Comment:
Let's put one of the valid maturity levels here:

> Valid values are: Working Draft, Candidate Recommendation, Recommendation.

See https://github.com/libp2p/specs/blob/master/00-framework-02-document-header.md#status-table.

I would recommend:

```suggestion
| 1A              | Candidate Recommendation                    | Active | r0, 2022-04-03  |
```

Reply:
Done, while also updating the date.